module-domain/src/main/kotlin/site/yourevents/guest/port/out/OwnerNicknamePersistencePort.kt

```diff
@@ -0,0 +1,7 @@
+package site.yourevents.guest.port.out
+
+import site.yourevents.guest.domain.Guest
+
+interface OwnerNicknamePersistencePort {
+    fun saveOwnerNickname(guest: Guest): Guest
```

Comment:
해당 인터페이스는 결국 GuestRepository에 의해 구현되기에, 해당 부분도 마찬가지로 네이밍 변경 제안드리는 바입니다..!! 😄 
```suggestion
interface GuestPersistencePort {
    fun save(guest: Guest): Guest
```

Reply:
마찬가지로 수정하였습니다!!